cinema_service/cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering your models here to make them accessible in the Django admin interface. Use `admin.site.register(YourModel)` to register each model.